### PR TITLE
Add `Stdout` exporter, timeout setting, better warning

### PR DESCRIPTION
**Pull request by SeungheonOh**

### CODE REVIEW

File: liqwid-script-export/src/ScriptExport/File.hs [hunk 1]
```diff
@@ -18,14 +18,19 @@ import System.Exit (die)
...
 runFile :: Builders -> FileOptions -> IO ()
 runFile builders options = do
   paramFile <- optional $ BS.readFile (view #param options)
   let param = paramFile >>= Aeson.decodeStrict'
       builder = view #builder options
 
+  maybe
+    (putStrLn "Warning: Failed to open the given parameter file, proceeding to generate without a parameter")
+    (const $ pure ())
+    param
```
Comment (emiflake) [inline liqwid-script-export/src/ScriptExport/File.hs]: Should probably use `when (isNothing param) $ putStrLn ...` instead here, reads a lot more humanly.

File: liqwid-script-export/src/ScriptExport/Options.hs [hunk 2]
```diff
@@ -61,14 +64,15 @@ data Options
...
--- | @since 2.0.0
+-- | @since 2.3.0
 data ServerOptions = ServerOptions
   { port :: Warp.Port
   -- ^ Which port to listen on.
   , enableCorsMiddleware :: Bool
   -- ^ Should we enable CORS middleware for debugging purposes?
   , cacheLifetime :: Int64
   -- ^ How long to keep cached items alive for.
+  , timeout :: Int
```
Comment (emiflake) [inline liqwid-script-export/src/ScriptExport/Options.hs:75]: Add a docstring here for what it does

File: liqwid-script-export/src/ScriptExport/Options.hs [hunk 3]
```diff
@@ -118,6 +136,25 @@ fileOpt =
...
+              <> Opt.short 'p'
+              <> Opt.metavar "PARAM"
+              <> Opt.help "Parameters to apply. If none is given, will expect stdin"
+          )
+      )
+    <*> Opt.strOption
+      ( Opt.long "builder"
+          <> Opt.short 'b'
+          <> Opt.metavar "BUILDER"
+          <> Opt.help "Bulider to use"
```
Comment (emiflake) [inline liqwid-script-export/src/ScriptExport/Options.hs]:
```suggestion
          <> Opt.help "Builder to use"
```

File: liqwid-script-export/src/ScriptExport/Options.hs [hunk 4]
```diff
@@ -147,6 +184,13 @@ serverOpt =
           <> Opt.value 3600
           <> Opt.help "How many seconds should the server keep cached results available for."
       )
+    <*> Opt.option
+      Opt.auto
+      ( Opt.long "timeout"
+          <> Opt.metavar "TIMEOUT"
+          <> Opt.value 6000
+          <> Opt.help "Script export server timeout in seconds. (default: 10 minuates)"
```
Comment (emiflake) [inline liqwid-script-export/src/ScriptExport/Options.hs]:
```suggestion
          <> Opt.help "Script export server timeout in seconds. (default: 10 minutes)"
```

**Review by emiflake — CHANGES_REQUESTED** [review]
Few minor things. Looks good overall though!

**Comment by emiflake** [comment]
Ah, also, please rebase on latest `main`, as it fixes the CI build.

**Comment by SeungheonOh** [comment]
Awesome, will do. Thank you!

**Review by emiflake — APPROVED** [review]
Good to go!

**Pull request merged**